tests/drivers/display/display_read_write/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2023 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Display Read/Write Test"
+
+source "Kconfig.zephyr"
+
+config DISPLAY_READ_WRITE_BUF_SRAM_EXT
```

Comment:
Why are you introducing this Kconfig, and then setting it to disabled for the only board being added here? Was the Kconfig originally intended to be enabled by this board?

If the Kconfig is no longer needed, I'd prefer that we drop it

Reply:
I introduced this Kconfig option so frame buffers can be allocated into external storage mediums, e.g. PSRAM, in case the SoC is running out of system RAM. The reason I have disabled that Kconfig in our board's configuration file is that the memory controller (MEMC) driver implementation has yet to be merged into main ([here](https://github.com/zephyrproject-rtos/zephyr/pull/68023) is the corresponding PR). Should I move this commit to the mentioned MEMC PR?

Reply:
This still needs to be dropped from this PR- it should be added in a follow up PR, once we have MEMC support for the Smartbond part

Reply:
Dropped